src/metabase/sync/sync_metadata/tables.clj

```diff
@@ -80,7 +80,7 @@
 (s/defn ^:private is-crufty-table? :- s/Bool
   "Should we give newly created TABLE a `visibility_type` of `:cruft`?"
   [table :- i/DatabaseMetadataTable]
-  (boolean (some #(re-find % (str/lower-case (:name table))) crufty-table-patterns)))
```

Comment:
str fns here?

Reply:
Oh yeah this is actually a bug fix I think. This logic wouldn't work if the locale was Turkish and the table names were upper case